scanner/src/main/kotlin/ScanResultsStorage.kt

```diff
@@ -103,6 +103,14 @@ abstract class ScanResultsStorage {
             properties["password"] = config.password
             properties["ApplicationName"] = "$ORT_NAME - $TOOL_NAME"
 
+            if (config.sslmode != "disabled") {
+                properties["ssl"] = "true"
+                properties["sslmode"] = config.sslmode
```

Comment:
Nit: This we could do in any case, and pull it out of the `if`, right?

Reply:
I have removed the `if` altogether and added a code comment.